agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/SnippetConfiguration.java

```diff
@@ -0,0 +1,32 @@
+          + "src: \"https://js.monitor.azure.com/scripts/b/ai.2.min.js\", // The SDK URL Source\n"
+          + "// name: \"appInsights\", // Global SDK Instance name defaults to \"appInsights\" when not supplied\n"
+          + "// ld: 0, // Defines the load delay (in ms) before attempting to load the sdk. -1 = block page load and add to head. (default) = 0ms load after timeout,\n"
+          + "// useXhr: 1, // Use XHR instead of fetch to report failures (if available),\n"
+          + "crossOrigin: \"anonymous\", // When supplied this will add the provided value as the cross origin attribute on the script tag\n"
+          + "// onInit: null, // Once the application insights instance has loaded and initialized this callback function will be called with 1 argument -- the sdk instance (DO NOT ADD anything to the sdk.queue -- As they won't get called)\n"
+          + "cfg: { // Application Insights Configuration\n"
+          + "connectionString: \"CONNECTION_STRING\"\n"
+          + "}});\n"
+          + "</script>\n";
```

Comment:
it would be nice if we can read this from a resource file, instead of inlining it into the Java code